opentelemetry-prometheus/tests/integration_test.rs

```diff
@@ -405,7 +405,12 @@ fn gather_and_compare(registry: prometheus::Registry, expected: String, name: &'
     let encoder = TextEncoder::new();
     let metric_families = registry.gather();
     encoder.encode(&metric_families, &mut output).unwrap();
-    let output_string = String::from_utf8(output).unwrap();
+
+    let output_string = if cfg!(windows) {
+        String::from_utf8(output).unwrap().replace('\n', "\r\n")
```

Comment:
We can build a util function for testing that takes a string and 
1. if os == windows do `replace('\n', "\r\n")`
2. if not return

Reply:
Updated.